custom-messaging-in-aboutwelcome-for-chrome-users-to-import.toml

```diff
@@ -0,0 +1,123 @@
+[metrics]
+# Most of these metrics "should" only increment on the first day the brower's used.
+# Report them weekly so that we get an early readout.
+# This gives us a bit of time to receive all of the telemetry from our onboarders
+# (expecting to receive every ping on day 0 is probably a losing game)
+# without making us wait a month for numbers.
+weekly = [
+    "opened_migration_wizard",
+    "imported_from_chrome",
+    "imported_from_any_browser",
+    "import_impression",
+    "theme_impression",
+    "exposed_treatment_a",
+    "exposed_treatment_b",
+    "exposed_control",
+]
+
+overall = [
+    "opened_migration_wizard",
+    "imported_from_chrome",
+    "imported_from_any_browser",
+    "import_impression",
+    "theme_impression",
+    "exposed_treatment_a",
+    "exposed_treatment_b",
+    "exposed_control",
+]
+
+# Key outcomes
+
+[metrics.opened_migration_wizard]
+select_expression = """COALESCE(SUM(
+    (SELECT SUM(value) FROM UNNEST(mozfun.hist.extract(payload.histograms.fx_migration_entry_point).values))
```

Comment:
this is "count of the number of observations in the histogram" and I should probably write a helper for it